### ISSUE TITLE
Removing number format from posts hash

### ISSUE THREAD

**Issue by ghost** [mybb/mybb#4652]
It may be more correct to remove the number format from the posts hash of 4 digits or more.

Like `#1195` instead of `#1,195`.

You can see how it looks with the link below:

👉 https://community.mybb.com/thread-169382-post-1383605.html#pid1383605

If you agree with me, I can send PR.

**Comment by Eldenroot** [comment]
Good idea, open a PR, MyBB team will maybe add a reply.

**Comment by Stefan-MyBB** [comment]
Take this huge thread for example: https://community.mybb.com/thread-123026-post-1380516.html#pid1380516)
`#28,247` is more readable than `#28247`

**Comment by ghost** [comment]
> Take this huge thread for example: https://community.mybb.com/thread-123026-post-1380516.html#pid1380516) `#28,247` is more readable than `#28247`

Yes like that. I think the current format is wrong according to the hash logic. (Github can be an example.) If we evaluate it as ID, it would be more appropriate without format. (My personal opinion.)

**Comment by dvz** [comment]
Note that in 1.9 the displayed post counter seems to be removed by default (but is still exposed in a [variable](https://github.com/mybb/mybb/blob/f3e4d92cc4c3c1d83ec72555e932686904bca46a/inc/functions_post.php#L677)), and the post date serves as the anchor text instead.

That seems to be a good choice as the counter doesn't take into account i.a. posts invisible to the viewing user, which may lead to confusion when referencing the displayed number, rather than a permalink.

**Comment by Sama34** [comment]
Also consider that hashing posts has nothing to do with any MyBB related (intended) feature; ideally MyBB should allow some kind of post mention similar to how Github does, and then we could establish some mechanic to make the result human friendly.

**Comment by Sama34** [comment]
I think my last post might make little sense.
